CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 [Full Changelog](https://github.com/folio-org/ui-search/compare/v1.9.0...v1.10.0)
 
 * Update of translations
+* Update "stripes" to 'v3.0.0', "stripes-core" to '4.0.0' and "react-intl" to '2.9.0'. Refs UISE-122.
```

Comment:
This line should be [here](https://github.com/folio-org/ui-search/pull/207/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR6).